content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: Request for Proposal - Javascript Development
+author: Athanasia M. Mowinckel
+date: '2022-03-28'
+slug: []
+categories: []
+tags: []
+lastmod: '2022-03-28T08:41:18+02:00'
+keywords: []
+description: ''
+comment: no
+toc: no
+autoCollapseToc: no
+contentCopyright: no
+reward: no
+mathjax: no
+---
+
+**R-Ladies wants to implement some new functionalities (using Javascript) for a re-implementation of their  website, whose source code may be found on [GitHub](https://github.com/rladies/website).**
+
+R-Ladies would like to invite you to prepare a proposal to accomplish the above task that includes timeline, cost, and deliverables. The following RFP includes a background of our organization and describes the purpose of the redesign, its desired functionality, and specific requests relating to the proposal. We understand that details may be subject to change upon vendor recommendation and/ or research of more optimal solutions. In your proposal, please feel free to suggest alternatives where noted.
+
+
+<!--more-->
+
+
+## Company Background
+R-Ladies, is a worldwide organisation whose mission is to promote gender diversity in the R community. 
+
+The R community suffers from an underrepresentation of minority genders (including but not limited to cis/trans women, trans men, non-binary, genderqueer, agender) in every role and area of participation, whether as leaders, package developers, conference speakers, conference participants, educators, or users (see recent stats).
+
+As a diversity initiative, the mission of R-Ladies is to achieve proportionate representation by encouraging, inspiring, and empowering people of genders currently underrepresented in the R community. R-Ladies’ primary focus, therefore, is on supporting minority gender R enthusiasts to achieve their programming potential, by building a collaborative global network of R leaders, mentors, learners, and developers to facilitate individual and collective progress worldwide.
+
+
+### Project Manager
+Athanasia Monika Mowinckel
+
+
+## Budget
+We ask prospective contractors to provide a preliminary budget for their proposal. This can be provided as hourly rates with estimated amounts of hours to complete the project. Development work can be unpredictable, and we understand that estimated hours to completion might diverge somewhat from proposal estimates. 
+
+## Timeline
+
+### RFP Release Date
+The date you’ve sent out the RFP. This lets vendors see how recent the request is.
```

Comment:
I thought vendor = people writing the code and R-Ladies = customer?

Reply:
I'm going to delete this part, I forgot to when I copy pasted! The release date is the date the blogpost goes live :)

Reply:
but yes, vendor is people writing the code, and we are the customer. So I think the intent here is to have a date that shows when the request was launched, so vendors can assess if this is a recent or old request.